app/android/app/build.gradle

```diff
@@ -31,7 +31,7 @@ if (keystorePropertiesFile.exists()) {
 }
 
```

Comment:
I am not failimar with gradle, gonna to trust this and test it after merging with your code..